packages/big-design/src/components/Worksheet/hooks/spec.tsx

```diff
@@ -0,0 +1,87 @@
+import { renderHook } from '@testing-library/react-hooks';
+import React from 'react';
+import { act } from 'react-test-renderer';
+
+import { Cell } from '../types';
+
+import { useEditableCell } from './useEditableCell';
+
+let mockUpdateItems = jest.fn();
+jest.mock('./useUpdateItems', () => ({
+  useUpdateItems: () => ({
+    updateItems: mockUpdateItems,
+  }),
+}));
+
+describe('useEditableCells', () => {
+  const cell: Cell<unknown> = { rowIndex: 0, columnIndex: 0, type: 'text', hash: 'category', value: 'Text' };
```

Comment:
Shouldn't this be `string`?
```suggestion
  const cell: Cell<string> = { rowIndex: 0, columnIndex: 0, type: 'text', hash: 'category', value: 'Text' };
```